uni/lib/view/home/widgets/restaurant_card.dart

```diff
@@ -1,13 +1,19 @@
 import 'package:flutter/material.dart';
 import 'package:provider/provider.dart';
 import 'package:uni/model/providers/lazy/restaurant_provider.dart';
-import 'package:uni/view/common_widgets/date_rectangle.dart';
 import 'package:uni/view/common_widgets/generic_card.dart';
 import 'package:uni/view/common_widgets/request_dependent_widget_builder.dart';
 import 'package:uni/view/common_widgets/row_container.dart';
-import 'package:uni/view/home/widgets/restaurant_row.dart';
+import 'package:uni/model/entities/meal.dart';
+import 'package:uni/model/entities/restaurant.dart';
+import 'package:uni/model/utils/day_of_week.dart';
+import 'package:uni/utils/drawer_items.dart';
+import 'package:uni/view/restaurant/widgets/restaurant_slot.dart';
 import 'package:uni/view/lazy_consumer.dart';
 
+final int weekDay = DateTime.now().weekday;
+final offset = (weekDay > 5) ? 0 : (weekDay - 1) % DayOfWeek.values.length;
```

Comment:
This is not working properly. Today is Saturday, so weekDay = 6 here, and offset is 0, making it Monday. In fact, the card does not show meals for today.

Reply:
Also, this should not be a global variable. This is a really fast computation, move it to a local variable where it is needed.